ext/tidy/tidy.stub.php

```diff
@@ -59,9 +62,13 @@ class tidy
     /** @param array|string $config_options */
     public function __construct(string $filename = UNKNOWN, $config_options = UNKNOWN, string $encoding = UNKNOWN, bool $use_include_path = false) {}
 
-    /** @return string|int|bool */
+    /**
+     * @return string|int|bool
+     * @alias tidy_getopt
+     */
     public function getOpt(string $option) {}
 
+    /** @alias tidy_clean_repair */
     public function cleanRepair(): bool {}
```

Comment:
Not related, but noticed that we have return types here, even though `tidy` is not final. Apparently only `tidyNode` is.

Reply:
Good catch! I'll fix this.